src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery.csproj

```diff
@@ -6,13 +6,13 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.RecoveryServices.SiteRecovery</PackageId>
 		<Description>Provides developers with libraries for the updated recovery services site recovery feature under Azure Resource manager.</Description>
-		<Version>1.1.0-preview</Version>
+		<Version>2.0.0-preview</Version>
```

Comment:
@avneeshrai since this is a preview version, please bump the minor version only i.e., `1.2.0-preview` please make sure the `AssemblyInfo.cs` is updated too